contrib/oc-get-customization.sh

```diff
@@ -0,0 +1,44 @@
+# shellcheck shell=bash
+#
+# This file is an example of how you might set up your environment to run the
+# console against an OpenShift cluster during development. To use it for
+# running bridge, do
+#
+# . contrib/oc-environment.sh
+# . contrib/oc-get-customization.sh
```

Comment:
It would be better to make this one script instead of two (even if this script calls the other).

Reply:
The problem with this is, oc-get-customization works only as kube:admin, while oc-environment works also with other users. And its only required to test customization locally, so I don't want to force anyone to use this but want provide a way to use it optionally.

We can improve this when we how we run the bridge with different users. Aka, after FF? Wdyt?

Reply:
Since this is only for local testing of the custom Dev catalog categories it could be a separate script. Would be good to update Readme with it.
@spadgett what do you think ?

Reply:
As @spadgett mentioned in the Slack discussion, lets leave this script out of the scope of this PR.